src/javascript/JContent/ContentRoute/ToolBar/FileModeSelector/FileModeSelector.jsx

```diff
@@ -1,62 +1,43 @@
 import React from 'react';
 import {useDispatch, useSelector} from 'react-redux';
 import {useTranslation} from 'react-i18next';
-import {filesgridSetGridMode, filesgridSetMode} from '../../ContentLayout/FilesGrid/FilesGrid.redux';
+import {filesgridSetMode} from '../../ContentLayout/FilesGrid/FilesGrid.redux';
 import JContentConstants from '../../../JContent.constants';
-import {Button, ViewComfy, ViewGrid, ViewList} from '@jahia/moonstone';
+import {Button, ViewGrid, ViewList} from '@jahia/moonstone';
 
 const localStorage = window.localStorage;
 
 const GRID = JContentConstants.mode.GRID;
 const LIST = JContentConstants.mode.LIST;
-const DETAILED = JContentConstants.gridMode.DETAILED;
-const LIST_VIEW = JContentConstants.gridMode.LIST;
-const THUMBNAIL = JContentConstants.gridMode.THUMBNAIL;
+const THUMBNAILS = JContentConstants.THUMBNAILS;
+const LIST_VIEW = JContentConstants.LIST;
```

Comment:
It would be much more clear to only use JContentConstants.mode.GRID and JContentConstants.mode.LIST everywhere instead of adding other constants. You can replace all usage of THUMBNAILS by JContentConstants.mode.GRID and LIST_VIEW by JContentConstants.mode.LIST